src/charm.py

```diff
@@ -332,6 +332,7 @@ def _configure_instance(self, container) -> bool:
             # Run mysqld for the first time to
             # bootstrap the data directory and users
             logger.debug("Initializing instance")
+            self._mysql.fix_data_dir()
```

Comment:
```
unit-mysql-k8s-0: 12:17:26 DEBUG unit.mysql-k8s/0.juju-log Initializing instance
unit-mysql-k8s-0: 12:17:26 ERROR unit.mysql-k8s/0.juju-log Uncaught exception while in charm code:
Traceback (most recent call last):
  File "/var/lib/juju/agents/unit-mysql-k8s-0/charm/./src/charm.py", line 739, in <module>
    main(MySQLOperatorCharm)
  File "/var/lib/juju/agents/unit-mysql-k8s-0/charm/venv/ops/main.py", line 439, in main
    framework.reemit()
  File "/var/lib/juju/agents/unit-mysql-k8s-0/charm/venv/ops/framework.py", line 840, in reemit
    self._reemit()
  File "/var/lib/juju/agents/unit-mysql-k8s-0/charm/venv/ops/framework.py", line 919, in _reemit
    custom_handler(event)
  File "/var/lib/juju/agents/unit-mysql-k8s-0/charm/./src/charm.py", line 449, in _on_mysql_pebble_ready
    if not self._configure_instance(container):
  File "/var/lib/juju/agents/unit-mysql-k8s-0/charm/./src/charm.py", line 335, in _configure_instance
    self._mysql.fix_data_dir()
TypeError: MySQL.fix_data_dir() missing 1 required positional argument: 'container'
unit-mysql-k8s-0: 12:17:26 ERROR juju.worker.uniter.operation hook "start" (via hook dispatching script: dispatch) failed: exit status 1
```

Reply:
Fixed in https://github.com/canonical/mysql-k8s-operator/pull/238/commits/ba735a9dae30844e0ff6869673b7a023345333ad